src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -65,7 +67,7 @@ public override void ExecuteCmdlet()
                         throw new ArgumentException(Resources.AzureProfileMustNotBeNull);
                     }
 
-                    if (!AzureSession.DataStore.FileExists(Path) || Force ||
+                    if (!AzureSession.DataStore.FileExists(Path) || Force.IsPresent ||
```

Comment:
Because this is not a similar situtation.  In this situation, we are not trying to determien the parameter set used, becasue there is only one parameter set.  Instead, we are checking file existence to give an overwte prompt, if a file will be overwritten